README.md

```diff
@@ -79,6 +79,24 @@ import XMTP
 let client = try await Client.create(account: account)
 ```
 
+### Creating a client from saved keys
+
+You can save your keys from the client via the `v1keys` property:
+
+```swift
+// Create the client with a `SigningKey` from your app
+let client = try await Client.create(account: account)
+
+// Save the keys somewhere safe
+let keys = client.v1keys
```

Comment:
Would be helpful to show to how to serialize these keys as well, since the most common use-case is going to be to store them somewhere which doesn't accept a class.